pkg/util/sets/ipnet.go

```diff
@@ -0,0 +1,119 @@
+/*
+Copyright 2016 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package sets
+
+import (
+	"net"
+	"strings"
+)
+
+type IPNet map[string]*net.IPNet
```

Comment:
I am not sure how I feel about having an open-coded set implementation among all the auto-generated ones, especially since it has a different form and slightly expanded API.

@lavalamp thoughts?  Other options could include putting this into `util/net` or autogenerating it, and forcing callers to call List() and then convert to strings themselves, or even standardizing StringList() in generated sets (printf("%v") or allow some customization in set-gen's input).